src/message-processor.js

```diff
@@ -1,50 +1,49 @@
-var fs = require("fs");
-var path = require("path");
-
-var config = require("../package.json").config || {};
-var factory = require("./response-factory");
-var stepRegistry = require("./step-registry");
-var customMessageRegistry = require("./custom-message-registry");
-var executor = require("./executor");
-var refactor = require("./refactor");
-var dataStore = require("./data-store-factory");
-var impl_loader = require("./impl-loader");
-var loader = require("./static-loader");
-var inspector = require("inspector");
-var fileUtil = require("./file-util");
-var customScreenshotRegistry = require("./custom-screenshot-registry");
-var logger = require("./logger");
+import fs, {readFileSync} from "node:fs";
+import path from "node:path";
+import factory from "./response-factory.js";
+import stepRegistry from "./step-registry.js";
+import customMessageRegistry from "./custom-message-registry.js";
+import executor from "./executor.js";
+import refactor from "./refactor.js";
+import dataStore from "./data-store-factory.js";
+import impl_loader from "./impl-loader.js";
+import loader from "./static-loader.js";
+import inspector from "node:inspector";
+import fileUtil from "./file-util.js";
+import customScreenshotRegistry from "./custom-screenshot-registry.js";
+import logger from "./logger.js";
+
+const config = JSON.parse(readFileSync("./package.json")) || {};
```

Comment:
I changed this as a workaround for lack of ability to import json on ES modules; and the lack of a proper ecma standard for doing so (there is a weird `assert {type: "json" }` or `with {type: "json" }` proposal in the works for years, but it's not standardised and eslint doesn't support it.

However I am not super confident in it having the same portability given using of `fs`.